src/screens/LiquidityPools/LiquidityPools.js

```diff
@@ -299,13 +300,13 @@ const LiquidityPoolsScreen = ({
     const poolStats = poolsStats[index];
 
     const { currentPrice } = poolStats;
-    const stakedAmountInFiat = convertUSDToFiat(currentPrice * poolStats.stakedAmount, rates, fiatCurrency);
+    const stakedAmountInFiat = convertUSDToFiat(currentPrice * poolStats.stakedAmount.toNumber(), rates, fiatCurrency);
     const formattedStakedAmount = formatFiat(stakedAmountInFiat, fiatCurrency);
     const tokenSymbol = pool.rewards[0].symbol;
 
     return (
       <TouchableOpacity onPress={() => goToPoolDashboard(pool)}>
-        {renderPoolListItem(pool, poolStats.stakedAmount, formattedStakedAmount)}
+        {renderPoolListItem(pool, poolStats.stakedAmount.toNumber(), formattedStakedAmount)}
```

Comment:
More `toNumber`s :-)

Reply:
This is only a rough calculation, no harm to be done here.